packages/elements/src/tree/managers/tree-manager.ts

```diff
@@ -356,9 +359,9 @@ export class TreeManager<T extends TreeDataItem> {
   }
 
   /**
-   * Calculates the checked stated of the item
+   * Return checked state of the item.
    * @param item Original data item
-   * @returns Checked state of the item
+   * @returns item checked state: `1` for checked, `0` for unchecked and `-1` for indeterminate
```

Comment:
this looks a bit odd but I guess the intention is to focus on Javascript developer because TS developers would use CheckedState type so it will return 'CHECKED', 'INDETERMINATE', 'UNCHECKED'. Right? I might suggest to revise it to more generic statement. For example, `@returns item checked state: checked, unchecked or indeterminate.`

Reply:
TypeScript users would be able to map this 3 numbers back to its enum value. However, JavaScript users won't be able to do so without looking at the source code if we don't list them out here.

Reply:
Yeah I see what you mean. Not strong feedback of mine here though. Happy to keep the same wording, my point is just make this more generic so it isn't too specific to any JS or TS but I agree that it could make the doc less useful too.